app/views/forms/show.html.erb

```diff
@@ -46,6 +37,8 @@
                                            sections: @task_list)
     %>
 
-    <%= govuk_button_link_to t("forms.delete_form"), delete_form_path(@form.id), warning: true %>
+
+    <%= govuk_button_link_to(t("forms.delete_form"), delete_form_path(@form.id), warning: true) unless @form.live?  %>
+
```

Comment:
Did you mean to add the extra whitespace here? If you did then feel free to ignore this